Readme.md

```diff
@@ -0,0 +1,35 @@
+# KOB PIU
+
+## Abstract
+
+KochiOrgBook public identity utility, an identity network for the city of Kochi. All the DIDs will be hosted upon the identity network. This network which is going to be the 1st layer of ToIp tecnology stack. The first layer is mainly for blockchains and distributed ledgers. The identity utility network is similar to that of Sovrin network.
+
+## Dependent Projects
+## Motivation
+
+As Individuals and Organizations use variety of credentials and documents for different purpose and we share those documents in internet. These credentials and documents serve as a unique identifiers which gives our communication addresses, ID Numbers and other personal informations. Sharing these personalized information in the Internet is not safe, attackers can get the information and can use them.  So we are here to introduce a new type of gloablly unique identifiers called DIDs which helps individuals and organizations to generate our own identifiers using systems we trust, and to prove control of those identifiers (authenticate) using cryptographic proofs (for example, digital signatures).
```

Comment:
"So we are here to introduce a new type of gloablly unique identifiers called DIDs" - This point is incorrect as in we are not the ones who introduced it, we are just using it.